systems/visma/validator.js

```diff
@@ -0,0 +1,142 @@
+      return error('Ingen ansettelsesforhold ble funnet i HRM', { employments: (hrm.employments || null) })
+    }
+
+    const positions = getPositions(employment)
+    if (!positions) {
+      if (user.expectedType === 'student') return warn('Ingen stillinger ble funnet i HRM', { employment, positions: (positions || null) })
+      return error('Ingen stillinger ble funnet i HRM', { employment, positions: (positions || null) })
+    }
+
+    const primaryPositions = positions.map(position => position['@isPrimaryPosition'] === 'true')
+    const activePrimaryPositions = primaryPositions.map(position => position.active)
+    const activePrimaryPosition = activePrimaryPositions.includes(true)
+
+    const activePositions = positions.map(position => position.active)
+    const activePosition = activePositions.includes(true)
+
+    // Sjekk at det finnes et aktivt ansettelsesforhold og minst én aktiv stilling
+    if (employment.active && activePrimaryPosition) {
+      if (user.expectedType === 'student') return warn('Fant aktivt ansettelsesforhold og stilling i HRM', { employment, positions })
+      return success('Fant aktivt ansettelsesforhold og stilling i HRM', { employment, positions })
+    }
+
+    // Fant kun et ansettelsesforhold
+    if (employment.active) {
+      // Krøss i taket om dette noen gang skjer, men..
+      if (!activePrimaryPosition && activePosition) {
+        if (user.expectedType === 'student') return warn('Fant et aktivt ansettelsesforhold i HRM, men ingen av de aktive stillingene er en hovedstilling', { employment, positions })
+        return error('Fant et aktivt ansettelsesforhold i HRM, men ingen av de aktive stillingene er en hovedstilling', { employment, positions })
+      }
+
+      if (user.expectedType === 'student') return warn('Fant et aktivt ansettelsesforhold i HRM, men ingen aktiv hovedstilling', { employment, positions })
+      return error('Fant et aktivt ansettelsesforhold i HRM, men ingen aktiv hovedstilling', { employment, positions })
+    }
+
+    // Fant kun aktiv(e) stilling(er)
+    if (activePrimaryPosition) {
+      const message = `Fant ${activePrimaryPosition.length > 1 ? 'flere aktive hovedstillinger' : 'én aktiv hovedstilling'}, men ikke noe ansettelsesforhold`
+      if (user.expectedType === 'student') return warn(message, { employment, positions })
+      return error(message, { employment, positions })
+    }
+
+    // Verken aktive stillinger eller ansettelsesforhold ble funnet
+    if (user.expectedType === 'student') return success('Det ble ikke funnet noe aktivt ansettelsesforhold i HRM', { employment, positions })
+    return error('Det ble ikke funnet noe aktivt ansettelsesforhold eller stillinger i HRM', { employment, positions })
+  }),
+  test('visma-03', 'Hovedstilling har korrekt kategori', 'Kontrollerer at hovedstillingen ikke har en kategori som er unntatt fra å få brukerkonto', () => {
```

Comment:
```suggestion
  test('visma-03', 'Ansettelsesforholdet har korrekt kategori', 'Kontrollerer at ansettelsesforholdet ikke har en kategori som er unntatt fra å få brukerkonto', () => {
```

Endrer fra 'stillingen' til 'ansettelsesforholdet', for å bruke konsekvente uttrykk hele veien